instrumentation/helpers-sql-obfuscation/README.md

```diff
@@ -0,0 +1,66 @@
+
+## Usage
+
+Add the gem to your instrumentation's gemspec file:
+
+```ruby
+# opentelemetry-instrumentation-your-gem.gemspec
+  spec.add_dependency 'opentelemetry-instrumentation-helpers-sql-obfuscation'
+```
+
+Add the gem to your instrumentation's Gemfile:
+
+```ruby
+# Gemfile
+
+group :test, :development do
+  gem 'opentelemetry-instrumentation-helpers-sql-obfuscation', path: '../helpers-sql-obfuscation'
+end
+```
+
+Make sure the `Instrumentation` class for your gem contains configuration options for:
+
+- `:db_statement`: the type of statement you want added as an attribute
+  Example: `option :db_statement, default: :include, validate: %I[omit include obfuscate]`
+- `:obfuscation_limit`: the length at which the obfuscated SQL string will be truncated.
+  Truncated SQL is not obfuscated. # TODO: Check this!
+  Example: `option :obfuscation_limit, default: 2000, validate: :integer`
+
+If you want to add support for a new adapter, update the following constants to include keys for your adapter:
+
+- `DIALECT_COMPONENTS`
+- `CLEANUP_REGEX`
+
+You must also add a new constant that calls the `generate_regex` method with your adapter's DIALECT_COMPONENTS that is named like `<ADAPTER>_COMPONENTS_REGEX`, such as: `MYSQL_COMPONENTS_REGEX`.
+
+Check [New Relic's SQL Obfuscation Helpers module][new-relic-obfuscation-helpers] to see if regular expressions for your adapter already exist.
+
+## Examples
+
+To obfuscate sql in your library:
+
+```ruby
+sql = OpenTelemetry::Instrumentation::Helpers::SqlObfuscation.obfuscate_sql(sql, config, :postgres).to_s
+```
+
+## How can I get involved?
+
+The `opentelemetry-instrumentation-helpers-sql-obfuscation` gem source is [on github][repo-github], along with related gems including `opentelemetry-pg` and `opentelemetry-trilogy`.
+
+The OpenTelemetry Ruby gems are maintained by the OpenTelemetry-Ruby special interest group (SIG). You can get involved by joining us on our [gitter channel][ruby-gitter] or attending our weekly meeting. See the [meeting calendar][community-meetings] for dates and times. For more information on this and other language SIGs, see the OpenTelemetry [community page][ruby-sig].
+
+## License
+
+The `opentelemetry-instrumentation-helpers-sql-obfuscation` gem is distributed under the Apache 2.0 license. See [LICENSE][license-github] for more information.
+
+[new-relic-obfuscation-helpers]: https://github.com/newrelic/newrelic-ruby-agent/blob/96e7aca22c1c873c0f5fe704a2b3bb19652db68e/lib/new_relic/agent/database/obfuscation_helpers.rb
+[repo-github]: https://github.com/open-telemetry/opentelemetry-ruby
+[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/LICENSE
+[ruby-sig]: https://github.com/open-telemetry/community#ruby-sig
+[community-meetings]: https://github.com/open-telemetry/community#community-meetings
+[ruby-gitter]: https://gitter.im/open-telemetry/opentelemetry-ruby
```

Comment:
@ericmustin - Is Ruby gitter still the preferred place to chat? I would've expected Slack, but I don't see it on the other READMEs.

Reply:
good catch, yea gitter is dead afaik long live slack, we can handle this change as a bulk change in a separate pr tho

Reply:
cool, I'll make an issue for it.

Reply:
https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/547